eventkit_cloud/ui/static/ui/app/components/StatusDownloadPage/StatusDownload.js

```diff
@@ -94,11 +99,17 @@ export class StatusDownload extends React.Component {
                     this.timer = null;
                     window.clearTimeout(this.timeout);
                     this.timeout = window.setTimeout(() => {
-                        this.props.getDatacartDetails(this.props.params.jobuid);
+                        this.props.getDatacartDetails(this.props.router.params.jobuid);
                     }, 270000);
                 }
             }
         }
+        if (nextProps.location !== this.props.location) {
```

Comment:
Thanks for tackling this bug Lars!
Does this comparison work correctly? I have not pulled down to test but im pretty sure location is an object so the comparison would be for the objects themselves and not the actual values within the object.

Reply:
It should work since the object itself gets replaced.